### ISSUE TITLE
Picture followups on WPTD nominees appreciation

### ISSUE THREAD

**Issue by abhansnuk** [wpmarketingteam/Marcomms-Rolling#48]

**Comment by nalininsbs** [comment]
Original emails all went out - confirmed by Larissa and Erica. Text / copy thanks to Erica, Hari, Larissa, Abha, (other names to be added).

**To do:**
An update is due for remainder who have not as yet replied. Will check-in with @murillol and Erica for the calendar of promotions for the polyglot nominations. Please do not close this card until we have done this to remind.

**Comment by murillol** [comment]
I don't quite remember what this task is about @nalininsbs , could you please remind me?

**Comment by abhansnuk** [comment]
This will be looked at during WCEU contributor day too as some of the nominees will be present. Some of them do not have images which is why the images and videos have not been able to be made. Meher had identified some follow-ups for WCEU too. @majadidit had offered to help with this too and offered space on GoDaddy area if needed.

**Comment by nalininsbs** [comment]
Copying over update provided on this in meetings and the drive:
- call for volunteers with video editing and photo editing made in Polyglots newsletter July 2022 and Marketing meeting July 2022. Note contributors assisting with this need access to software that does not leave a commercial watermark on the video or photos. This can also distort the video or photo from an accessibility perspective. Need to check all subtitles must be in fonts that are clear and have sufficient color contrast.
- waiting on response from a couple of people who have submitted videos so that these can be completed and provided for future Polyglots newsletters
- once WPTD set for 2022, then these can also be added as links for this to the relevant media. 

Was there anything else @abhansnuk and @meherbala from your updates over this? Thank you for following up and assisting polyglots.

**Comment by nalininsbs** [comment]
Do we have a link and schedule for WordPress Translation Day 2022 promotions? This will allow us to move this ticket forward. If there is not going to be a global event this year, then the schedule can be adjusted in line with the polyglot newsletter.

Added to the GitHub ticket check-ins for September 7. 2022.

**Comment by abhansnuk** [comment]
This was not needed for the lower key WPTD in 2022 due to resources and volunteers. More on this and wider short pieces on polyglots will be followed up as part of polyglot outreach work and newsletter, and discussion in marketing channel with polyglots. It will be refocused on in early 2023. 

This task will then be updated/ merged into a new task. Putting this back into to do column for now.

**Comment by abhansnuk** [comment]
There have been some new volunteers in marketing and some new people in polyglots who have also joined marketing who are interested in this.

**Comment by nalininsbs** [comment]
Moving this to other teams column.
